spec/models/chargeback_container_project_spec.rb

```diff
@@ -238,4 +244,60 @@
       expect(subject.fixed_compute_1_cost).to be_within(0.01).of(@hourly_rate * @metric_size)
     end
   end
+
+  context "tagged project" do
+    before do
+      @options[:interval] = "monthly"
+      @options[:tag] = "/managed/environment/prod"
+
+      tz = Metric::Helper.get_time_zone(@options[:ext_options])
+      ts = Time.now.in_time_zone(tz)
+      time     = ts.beginning_of_month.utc
+      end_time = ts.end_of_month.utc
+
+      while time < end_time
+        @live_group.metric_rollups << FactoryGirl.create(:metric_rollup_vm_hr,
+                                                         :timestamp                => time,
+                                                         :cpu_usage_rate_average   => @cpu_usage_rate,
+                                                         :derived_vm_numvcpus      => @cpu_count,
+                                                         :derived_memory_available => @memory_available,
+                                                         :derived_memory_used      => @memory_used,
+                                                         :net_usage_rate_average   => @net_usage_rate,
+                                                         :parent_ems_id            => @ems.id,
+                                                         :tag_names                => "",
+                                                         :resource_name            => @live_group.name)
+
+        @disconnected_group.metric_rollups << FactoryGirl.create(:metric_rollup_vm_hr,
+                                                                 :timestamp                => time,
+                                                                 :cpu_usage_rate_average   => @cpu_usage_rate,
+                                                                 :derived_vm_numvcpus      => @cpu_count,
+                                                                 :derived_memory_available => @memory_available,
+                                                                 :derived_memory_used      => @memory_used,
+                                                                 :net_usage_rate_average   => @net_usage_rate,
+                                                                 :parent_ems_id            => @ems.id,
+                                                                 :tag_names                => "",
+                                                                 :resource_name            => @live_group.name)
```

Comment:
Same rollups as "Monthly", same up to times as "Daily".  Extract helper function?